cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -304,6 +305,7 @@ func runAgent(ctx context.Context,
 		RotationInterval:    ag.Config.Agent.LogfileRotationInterval,
 		RotationMaxSize:     ag.Config.Agent.LogfileRotationMaxSize,
 		RotationMaxArchives: ag.Config.Agent.LogfileRotationMaxArchives,
+		LogWithTimezone:     "",
```

Comment:
What would happen if the LogWithTimeZone is empty string? Would we always want to set UTC Timezone as previous behavior or would we consider moving to Local Timezone to match consistency Customer's Timezone?

Reply:
Using UTC was the existing behavior, using an empty string will result in fall back to UTC, hence maintain the same behavior.